services/kuber/server/nodes/nodes.go

```diff
@@ -129,11 +134,10 @@ func (d *Deleter) deleteFromEtcd(kc kubectl.Kubectl) error {
 	for _, nodeName := range d.masterNodes {
 		for _, etcdPodInfo := range etcdPodInfos {
 			if nodeName == etcdPodInfo.nodeName {
-				log.Info().Msgf("Deleting etcd member %s, with hash %s ", etcdPodInfo.nodeName, etcdPodInfo.memberHash)
+				log.Debug().Msgf("Deleting etcd member %s, with hash %s ", etcdPodInfo.nodeName, etcdPodInfo.memberHash)
```

Comment:
Same here - `Info` level

Reply:
This log is more useful when debugging than for a user IMO. It shows quite an "under the hood" message, and it also misses the information about which cluster are those nodes from. Thus I used the `debug` level while keeping the info about deleting a master node in the `info` level. :)

Reply:
> Thus I used the debug level while keeping the info about deleting a master node in the info level. :)
Great thank you!
I think on this you might be right. You convinced me 👍